### ISSUE TITLE
Total size of the repository makes cloning to submit a PR to add a new entry difficult

### ISSUE THREAD

**Issue by avivace** [gbdev/database#229]

**Comment by pinobatch** [comment]
Would shallow cloning help any? Or is even the current state of the repository impractically big?

**Comment by tobiasvl** [comment]
LFS?

**Comment by avivace** [comment]
See also: https://github.com/gbdev/homebrewhub/issues/105

**Comment by RetroEdit** [comment]
> [@RetroEdit](https://github.com/RetroEdit) Having the binaries in another repo would still mean you have to clone that repo to send a PR, no?

Yeah, if you're modifying a binary, but it wouldn't be necessary for making a metadata-only change.

To me, there's some argument for separating them just for the principle of it. The metadata should never have license issues, but binaries that are only freeware and not freely licensed can end up being removed if people request it.

But maybe it's not worth it at this point if the metadata and binaries are designed to be intimately connected with continuous sync.
